prog/vm/nexus.rb

```diff
@@ -226,15 +215,14 @@ def create_storage_volume_records(vm_host)
 
       spdk_installation_id =
         allocate_spdk_installation(
-          vm_host.spdk_installations,
-          use_bdev_ubi: volume["use_bdev_ubi"]
+          vm_host.spdk_installations
         )
 
       VmStorageVolume.create_with_id(
         vm_id: vm.id,
         boot: volume["boot"],
         size_gib: volume["size_gib"],
-        use_bdev_ubi: volume["use_bdev_ubi"],
+        use_bdev_ubi: SpdkInstallation[spdk_installation_id].supports_bdev_ubi?,
```

Comment:
nit: If `allocate_spdk_installation` returns `rand_choice` instead of `rand_choice.id` you can use it directly.
```suggestion
        use_bdev_ubi: spdk_installation.supports_bdev_ubi?,
```

Reply:
Keeping as it is as it looks a bit more explicit to me.